packages/core/lib/commands/console/run.js

```diff
@@ -1,10 +1,16 @@
 module.exports = async function (options) {
-  const Config = require("@truffle/config");
   const { Console, excludedCommands } = require("../../console");
   const { Environment } = require("@truffle/environment");
-
-  const config = Config.detect(options);
   const commands = require("../commands");
+  const loadConfig = require("../../loadConfig");
+
+  if (options.url && options.network) {
+    throw new Error(
+      "'url' and 'network' options should not be specified together"
```

Comment:
Can this be changed to "'url' and 'network' are mutually exclusive options"?

Reply:
Sure, I will change that. Thanks @cds-amal